### PR TITLE
Music mile 1

### PR DESCRIPTION
**Pull request by jasonfelice**
## Milestone: 1

- [x] Created `MusicAlbum` class
- [x] Created `Genre` class
- [x] Implemented `add_item method` in `Genre` class
- [x] Implemented `can_be_archived?` in `MusicAlbum` class

### CODE REVIEW

File: base/genre.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.add_genre(self)
```
Comment (rolandfbk) [inline base/genre.rb:12]: I can see that the add_genre method is applied to the item. Don't you think that it should also be set as an accessor on the Item class?

**Review by rolandfbk — COMMENTED** [review]
Hello @jasonfelice, I left a comment for you. I'm just suggesting. I don't know if I am right about it. Please try to check

File: base/genre.rb (hunk 1 above)
Comment (jasonfelice) [reply base/genre.rb:12]: Yes, I have created a `attr_accessor` for `genre`

Comment (rolandfbk) [reply base/genre.rb:12]: Ok thank you

**Review by rolandfbk — APPROVED** [review]

**Pull request merged**